src/ORM/DataObject.php

```diff
@@ -3819,6 +3819,32 @@ public function requireTable()
                         'columns' => [$childField],
                     ],
                 ];
+                // Add index for sort. MySQL (and probably others) can only use a single index at a time,
```

Comment:
Are you sure this is correct? Won't it do an index_merge if there's two or more indexes? (which is less efficient than a composite index)?

Reply:
Maybe I'm behind the times, I don't know what `index_merge` is. But it certainly used to be true.
In any case, if the composite index is more efficient then lets keep the composite index.